src/DynamoCore/Services/StabilityTracking.cs

```diff
@@ -100,6 +118,7 @@ public static void WriteUptimeBeat(TimeSpan timespan)
         /// </summary>
         public static void Startup()
         {
+            StabilityUtils.IsLastShutdownClean = IsLastShutdownClean();
```

Comment:
Got it, I think your intent was to store the `IsLastShutdownClean` so that it won't be modified at all, os that's good.